added 4.4.41.0 issue (#4945)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/rooms/known-issues.md

|  Device updated to unsupported version of Windows 10   |    Windows 10 device updated from version 1803 to version 1809, which is not supported. The supported version is 1903. |   This can happen if the [Group Policy or MDM setting for DeferFeatureUpdatesPeriodinDays](https://docs.microsoft.com/windows/deployment/update/waas-configure-wufb) setting, which lets you defer feature updates for a specified number of days, is set to the maximum of 365 days. <br><br> Windows 10 version 1809 isn't supported with Microsoft Teams Rooms, while version 1903 is supported. However, as of March 27, 2020, version 1809 is over 365 days old. If this setting isn't changed, Windows attempts to install version 1809, which may cause issues with Microsoft Teams Rooms.<br><br>To avoid this situation, **remove** any Group Policy or MDM setting for deferring updates. This allows Windows to update to the latest, supported OS version. <br><br>**IMPORTANT** The Group Policy or MDM setting must be **removed** (left unconfigured) and **not set to 0**. If the policy is set to 0, Windows takes the latest available version which may not be supported. |  None |

> | Application not launching |  After update to application version 4.4.41.0, system boots to black screen and may get stuck at black screen or goes to logon screen after few minutes. | Follow steps in this [guide](https://aka.ms/mtrapplaunchguide) to fix this issue  | None |

Reviewer: ```suggestion
| Application not launching |  After update to application version 4.4.41.0, the system boots to a black screen and may get stuck at the black screen or goes to the logon screen after few minutes. | Follow the steps in this [guide](https://aka.ms/mtrapplaunchguide) to fix this issue.  | None |
```

> | Application not launching |  After update to application version 4.4.41.0, system boots to black screen and may get stuck at black screen or goes to logon screen after few minutes. | Follow steps in this [guide](https://docs.microsoft.com/microsoftteams/troubleshoot/teams-administration/teams-rooms-app-wont-start-after-update) to fix this issue  | None |

Reviewer 2: ```suggestion
| Application not launching |  After updating to application version 4.4.41.0, the system boots to black screen where it may get stuck, or go to the logon screen after few minutes. | Follow the steps in [Microsoft Teams Rooms application does not start after update to version 4.4.41.0](https://docs.microsoft.com/microsoftteams/troubleshoot/teams-administration/teams-rooms-app-wont-start-after-update) to fix this issue.  | None |
```

|  Device updated to unsupported version of Windows 10   |    Windows 10 device updated from version 1803 to version 1809, which is not supported. The supported version is 1903. |   This can happen if the [Group Policy or MDM setting for DeferFeatureUpdatesPeriodinDays](https://docs.microsoft.com/windows/deployment/update/waas-configure-wufb) setting, which lets you defer feature updates for a specified number of days, is set to the maximum of 365 days. <br><br> Windows 10 version 1809 isn't supported with Microsoft Teams Rooms, while version 1903 is supported. However, as of March 27, 2020, version 1809 is over 365 days old. If this setting isn't changed, Windows attempts to install version 1809, which may cause issues with Microsoft Teams Rooms.<br><br>To avoid this situation, **remove** any Group Policy or MDM setting for deferring updates. This allows Windows to update to the latest, supported OS version. <br><br>**IMPORTANT** The Group Policy or MDM setting must be **removed** (left unconfigured) and **not set to 0**. If the policy is set to 0, Windows takes the latest available version which may not be supported. |  None |

> | Application not launching |  After update to application version 4.4.41.0, system boots to black screen and may get stuck at black screen or goes to logon screen after few minutes. | Follow steps in this [guide](https://docs.microsoft.com/en-us/microsoftteams/troubleshoot/teams-administration/teams-rooms-app-wont-start-after-update) to fix this issue  | None |

Reviewer 2: ```suggestion
| Application not launching |  After update to application version 4.4.41.0, system boots to black screen and may get stuck at black screen or goes to logon screen after few minutes. | Follow steps in this [guide](https://docs.microsoft.com/microsoftteams/troubleshoot/teams-administration/teams-rooms-app-wont-start-after-update) to fix this issue  | None |
```
